lib/commands/web.js

```diff
@@ -41,8 +43,10 @@ extensions.getSafariIsIphoneX = _.memoize(async function getSafariIsIphone () {
     const script = 'return {height: window.screen.availHeight, width: window.screen.availWidth};';
     const {height, width} = await this.execute(script);
     // check for the correct height and width
-    return (height === IPHONE_X_HEIGHT && width === IPHONE_X_WIDTH) ||
-           (height === IPHONE_X_WIDTH && width === IPHONE_X_HEIGHT);
+    const [portraitHeight, portraitWidth] = height > width ? [height, width] : [width, height];
+    return (portraitHeight === IPHONE_X_HEIGHT && portraitWidth === IPHONE_X_WIDTH) ||
```

Comment:
`return [IPHONE_X_HEIGHT, IPHONE_XR_HEIGHT].includes(portraitHeight) &&  [IPHONE_XR_WIDTH, IPHONE_XR_WIDTH].includes(portraitHeight)` would be shorter